Compose/NCSA/docker-compose.yml

```diff
@@ -0,0 +1,182 @@
+version: "3.4"
+
+services:
+
+  queue:
+    image: lsstts/develop-env:b22
+    container_name: queue
+    environment:
+      - LSST_DDS_DOMAIN=auxtelpath
+    volumes:
+      - /home/saluser/develop:/home/saluser/develop
+      - ./queue_setup.sh:/home/saluser/.setup.sh
+#      - ./ospl.xml:/opt/OpenSpliceDDS/V6.9.0/HDE/x86_64.linux/etc/config/ospl.xml
+    stdin_open: true
+    tty: true
+    networks:
+      test-stand-network:
+        ipv4_address: "141.142.238.100"
+
+  script:
+    image: lsstts/develop-env:b22
+    container_name: script
+    environment:
+      - LSST_DDS_DOMAIN=auxtelpath
+    volumes:
+      - /home/saluser/develop:/home/saluser/develop
+      - ./request_script_setup.sh:/home/saluser/.setup.sh
+#      - ./ospl.xml:/opt/OpenSpliceDDS/V6.9.0/HDE/x86_64.linux/etc/config/ospl.xml
+    stdin_open: true
+    tty: true
+    networks:
+      test-stand-network:
+        ipv4_address: "141.142.238.150"
+
+#  watcher:
```

Comment:
Why is the watcher turned off?

Reply:
Until we get LOVE setup and running it is not that useful to have the watcher running..

Reply:
OK. This is good to go.